StPetersburg/InEtn30521218.xml

```diff
@@ -0,0 +1,307 @@
+                           <lb/>ቱነ፡ <persName ref="PRS12868WaldaA">ወልደ፡ አምላክ፡</persName> ወምስለ፡
+                           <lb/>ለዓለመ፡ ዓለም፡ አሜን፡ ወ
+
+                         </incipit>
+                       </msItem>
+                       <msItem xml:id="ms_i1.3.2">
+                         <locus target="#58r"></locus>
+                         <title type="complete" ref="LIT1558Matthew#TituliMatthew"></title>
+                         <textLang mainLang="gez"></textLang>
+                       </msItem>
+                     </msItem>
+
+                     <msItem xml:id="ms_i1.4">
+                       <locus from="58r" to="83r"></locus>
+                       <title type="complete" ref="LIT1882MarkGo"></title>
+                       <textLang mainLang="gez"></textLang>
+
+                       <msItem xml:id="ms_i1.4.1">
+                         <locus from="58r" to="82r"></locus>
+                         <title type="complete" ref="LIT2711Mark" xml:lang="gez">ብስራተ፡ አብ፡ ቅዱስል ማቴዎስ፡ ማርቆስ፡</title>
+                         <textLang mainLang="gez"></textLang>
+                          <note>Organized according to days of week. The beginning of the text is built up in a sort of a puzzle.</note>
+                          <incipit xml:lang="gez">
+                            <lb/>ብስራተ፡ አብ፡ ቅዱስ፡ ማር
+                            <lb/>ቆስ፡ በረከቱ፡ ይኩን፡ ምስ
+                            <lb/>ሌነ፡ ለዓለመ፡ ዓለም፡ አሜ
+                            <cb/>፩ እም፸ወ፪ አርድእት፡
+                            <lb/>በረከቱ፡ ይኩን፡ ምስሌነ፡
+                            <lb/>ለዓለመ፡ ዓለም፡ አሜን፡ ወ
+                          </incipit>
+                       </msItem>
+                       <msItem xml:id="ms_i1.4.2">
+                         <locus from="82v" to="83r"></locus>
+                         <title type="complete" ref="LIT1882MarkGo#TituliMark"></title>
+                         <textLang mainLang="gez"></textLang>
+                       </msItem>
+
+                     </msItem>
+
+                     <msItem xml:id="ms_i1.5">
+                       <locus from="83r" to="125r"></locus>
+                       <title type="complete" ref="LIT1812GospelLuke"></title>
+                       <textLang mainLang="gez"></textLang>
+
+                       <msItem xml:id="ms_i1.5.1">
+                         <locus from="83r" to="84r"></locus>
+                         <title type="complete" ref="LIT1812GospelLuke#TituliLuke"></title>
+                         <textLang mainLang="gez"></textLang>
+                       </msItem>
+                       <msItem xml:id="ms_i1.5.2">
+                         <locus from="85r" to="125r"></locus>
+                         <title type="complete" ref="LIT2713Luke" xml:lang="gez">ብስራተ፡ አብ፡ ሉቃስ፡</title>
+                         <textLang mainLang="gez"></textLang>
+                         <note>Organized according to days of week. The beginning of the text is built up in a sort of a puzzle.</note>
```

Comment:
the description is puzzling to me, but it probably is given like this in the catalogue?

Reply:
yes, it is